Model/Service/DumpCmsDataService.php

```diff
@@ -49,12 +49,12 @@ public function __construct(
         $this->categoryList = $categoryList;
     }
 
-    public function execute(array $types, ?array $identifiers)
+    public function execute(array $types, ?array $identifiers, ?bool $removeAll)
```

Comment:
Remove `?bool`, only `bool` is needed since we need to have boolean variable at this point.